src/modules/ROOT/pages/rfc/envoy/layer-graph.adoc

```diff
@@ -0,0 +1,61 @@
+= Graph layer
+
+The graph layer helps us determine the *processing order*, keeping in mind the *dependency constraints* put in place by the resource nodes.
+
+== Graph nodes
+
+A graph node is any structure that conforms to the `envoy.Node` interface:
+
+[source,go]
+----
+Node interface {
+  // Matches checks if the node matches the given resources and **any** of the identifiers.
+  Matches(resource string, identifiers ...string) bool
+
+  // Identifiers returns a set of values that identify the node.
+  //
+  // The identifiers may **not** be unique across all resources, but
+  // they **must** be unique inside a given resource.
+  Identifiers() NodeIdentifiers
+
+  // Resource returns the Corteza resource identifier that this node handles
+  Resource() string
```

Comment:
Should be ResourceType() as it does not point to a specific resource.

Reply:
Plus, string does not be Corteza resource identifier... 
It needs to be something that can be understood & handled by encoder, correct?